schema.graphql

```diff
@@ -314,20 +291,11 @@ type SettingsNode implements Node {
   logoUrl: String
   lightLogoUrl: String
 
-  """
-  The ID of the object.
-  """
+  """The ID of the object."""
   id: ID!
   phase: Phase!
   startText: String
   managerOverallReviewText: String
-  selfReviewSahabinessHelpModalText: String
```

Comment:
what happens to this

Reply:
we need this

Reply:
you removed all these lines 
they had no usage ?